src/script/team/TeamState.ts

```diff
@@ -82,10 +84,6 @@ export class TeamState {
 
     this.supportsLegalHold = ko.observable(false);
 
-    this.userState.isTeam = this.isTeam;
-    this.userState.teamMembers = this.teamMembers;
-    this.userState.teamUsers = this.teamUsers;
-
```

Comment:
Those are weird assignement from one observable to another one. Let's use the single source of truth there and only consume the `TeamState` when we need to know who are the team members, for example